toolchest_client/tools/demucs.py

```diff
@@ -0,0 +1,30 @@
+"""
+toolchest_client.tools.demucs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the Demucs implementation of the Tool class.
+"""
+from . import Tool
+from toolchest_client.files import OutputType
+
+
+class Demucs(Tool):
+    """
+    The Demucs Omega implementation of the Tool class.
```

Comment:
typo: is it Demucs Omega or just Demucs?

Reply:
That is indeed a typo